skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkikoProperties.kt

```diff
@@ -5,8 +5,13 @@ internal object SkikoProperties {
     val vsyncEnabled: Boolean by property("skiko.vsync.enabled", default = true)
 
     val fpsEnabled: Boolean by property("skiko.fps.enabled", default = false)
-    val fpsCount: Int by property("skiko.fps.count", default = 300)
-    val fpsProbability: Double by property("skiko.fps.probability", default = 0.97)
+    val fpsPeriodSeconds: Double by property("skiko.fps.periodSeconds", default = 2.0)
+    val fpsLongFramesShow: Boolean by property("skiko.fps.longFrames.show", default = false)
+
+    /**
+     * If the property isn't defined, but skiko.fps.longFrames.show is true, this property will be 1.5 * (1000 / displayRefreshRate)
+     */
+    val fpsLongFramesMillis: Double? by property("skiko.fps.longFrames.millis", default = null)
```

Comment:
Why not write it here explicitly, i.e 
```
val fpsLongFramesMillis: Double? by lazy {
    val millis = if (fpsLongFramesShow) 1.1 else null
    property("skiko.fps.longFrames.millis", default = millis) 
}

Reply:
I think property itself should be simple and don't contain any complex logic. 
It is FpsCounter's job to check this.

Reply:
Well, isn’t that a definition of this property? For other it’s getting from JVM dictionary, for that one - somewhat trickier computation.

Reply:
'null' here means we should use automatic detection of fpsLongFramesMillis.

Without this logic we can define `fpsLongFramesMillis: Double` instead of `fpsLongFramesMillis: Double?`

In this case, code like this:
```
val fpsLongFramesMillis: Double by lazy {
    val millis = if (fpsLongFramesShow) 1.1 else 0.0
    property("skiko.fps.longFrames.millis", default = millis) 
}
```
Will be strange (why 0.0?)

We can do here not only parsing, but also write some domain structure like this:
```
object SkikoProperties {
    sealed class FpsLongFrames {
       class Auto: FpsLongFrames 
       class Manual: FpsLongFrames 
    }

    class Fps {
           val fpsLongFrames: FpsLongFrames?
    }

    val fps: Fps?
}
```

But it is more complex.

Anyway, I think it is not so important :)

I changed comments inside SkikoPoperties, it is slightly better now.

Reply:
OK, as you like.